packages/workers-shared/router-worker/src/worker.ts

```diff
@@ -95,6 +123,66 @@ export default {
 				}
 			}
 
+			const routeToUserWorker = async ({
+				asset,
+			}: {
+				asset: "static_routing" | "none";
+			}) => {
+				if (!config.has_user_worker) {
+					throw new Error(
+						"Fetch for user worker without having a user worker binding"
+					);
+				}
+				if (eyeballConfig.limitedAssetsOnly) {
+					analytics.setData({ userWorkerFreeTierLimiting: true });
+					return new Response(limitedResponse, {
+						status: 429,
+						headers: {
+							"Content-Type": "text/html",
+						},
+					});
+				}
+				analytics.setData({ dispatchtype: DISPATCH_TYPE.WORKER });
+				userWorkerInvocation = true;
+				return env.JAEGER.enterSpan("dispatch_worker", async (span) => {
+					span.setTags({
+						hasUserWorker: true,
+						asset: asset,
+						dispatchType: DISPATCH_TYPE.WORKER,
+					});
+
+					if (shouldBlockNonImageResponse) {
```

Comment:
We can actually move the computation of this boolean in here too I think, which would save us having to compute it if we never need to hit the user worker.

Reply:
yes, although the only difference that would change is that the analytics for `abuseMitigationURLHost` would not get set for requests that don't go to the asset worker. i think for the most part that's fine, however, since that's all we're interested in. making that change